packages/components/src/Autocomplete/Autocomplete.tsx

```diff
@@ -21,6 +25,11 @@ interface AutocompleteProps {
    */
   readonly placeholder: string;
 
+  /**
+   * Adjusts the input text box to either have small or large height.
+   */
+  readonly size?: FormFieldProps["size"];
```

Comment:
Since this is optional we should specify what it defaults to. Probably `base`?

Reply:
Just as a heads up we don't specify input text's default size.

Reply:
Ah, that's one of the old ones then. 😢 Don't worry about it here then.